__tests__/app.test.js

```diff
@@ -175,3 +175,81 @@ describe("GET api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("POST api/articles/:article_id/comments", () => {
+  test("Status-201: responds with an object containing the newly posted comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "Lovely article!",
+      created_at: new Date(),
```

Comment:
the request body should accept an object with the properties username + body, not username, body + created_at - this should be handled by the backend!